test/test_session.py

```diff
@@ -392,13 +404,30 @@ def test_cursor(self):
         coll = client.pymongo_test.collection
         coll.insert_many([{} for _ in range(1000)])
 
+        def lambda_find(session):
```

Comment:
This naming schema is a bit odd since none of these are "lambdas". How about we call them "find", "distinct", etc...?